tests/packaged_modules/test_imagefolder.py

```diff
@@ -17,6 +17,24 @@ def cache_dir(tmp_path):
     return str(tmp_path / "imagefolder_cache_dir")
 
 
+@pytest.fixture
+def data_dir_with_labels(tmp_path, image_file):
```

Comment:
One nit for consistent fixture names in this test module

```suggestion
def data_files_with_no_metadata(tmp_path, image_file):
```

Reply:
You also need to update the fixture name in the test